plugins/turing/main.go

```diff
@@ -0,0 +1,18 @@
+package main
```

Comment:
Refactored Experiment Engine plugin folder structure:
- Introduced `config`, `manager` and `runner` packages
- Removed `cmd` package

Reply:
Why is the `cmd` package removed? It's still useful for development.